backend/src/main/java/com/bakdata/conquery/models/query/queryplan/filter/SingleColumnEventFilterNode.java

```diff
@@ -0,0 +1,26 @@
+package com.bakdata.conquery.models.query.queryplan.filter;
+
+import java.util.Set;
+
+import javax.validation.constraints.NotNull;
+
+import com.bakdata.conquery.models.datasets.Column;
+import com.bakdata.conquery.models.identifiable.ids.specific.TableId;
+import lombok.Getter;
+import lombok.Setter;
+
+public abstract class SingleColumnEventFilterNode<FE_TYPE> extends EventFilterNode<FE_TYPE> {
+
+	public SingleColumnEventFilterNode(Column column, FE_TYPE filterValue) {
+		super(filterValue);
+		this.column = column;
+	}
+
+	@NotNull @Getter @Setter
+	private Column column;
+
+	@Override
+	public void collectRequiredTables(Set<TableId> requiredTables) {
```

Comment:
Eigentlich könnten wir hier auch die Table zurückgeben anstatt der ID. Der Queryplan existiert ja nur noch auf dem Worker und muss nicht mehr serialisiert/deserializiert werden.
Können wir uns mal überlegen, bei einiges Selects wird eine Table zu einer Id umgewandelt bevor der Aggregator erzeugt wird. Manche haben eine statische Table Id ALL_ID-Table.

Reply:
Mache ich in einem separaten PR